### DISCUSSION TITLE
Status stuck in this status for over 3 days: Enforce HTTPS — Not yet available for your site because the certificate has not finished being issued.

### DISCUSSION THREAD

**Discussion by alefaeronautics** [community/community#32941] [Other Feature Feedback, Questions, & Ideas]
Hi. I did all DNS correctly (and Github confirms it), verified that I owe domain about 3 days ago. But my https status is still stuck at: Enforce HTTPS — Not yet available for your site because the certificate has not finished being issued.
What is going on?

Thanks,
Jim.

**Comment by yoannchaudet** [comment]
Hello, please share the name of your domain here or open a [support ticket](https://support.github.com/request/pages?tags=dotcom-footer%2Chubberfy_pages) to do so.

**Reply by zufardhiyaulhaq** [reply]
Hi @yoannchaudet happen with me today.
<img width="992" alt="image" src="https://user-images.githubusercontent.com/11990726/192077414-1d9c6a38-126c-4e52-94b1-1fdea937360c.png">

**Reply by yoannchaudet** [reply]
Hello, I see you certificate was approved some time ago. You can now enforce HTTPS on your domain if you wish to.
